src/test/java/attendance/domain/AttendanceManagerTest.java

```diff
@@ -0,0 +1,315 @@
+package attendance.domain;
+
+import attendance.utility.DateGenerator;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Stream;
+
+import static attendance.domain.AttendanceStateType.ABSENCE;
+import static attendance.domain.AttendanceStateType.ATTENDANCE;
+import static attendance.domain.AttendanceStateType.LATE;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatIllegalArgumentException;
+
+class AttendanceManagerTest {
+
+    private DateGenerator dateGenerator;
+    private AttendanceManager attendanceManager;
+
+    @BeforeEach
+    void beforeEach() {
+        dateGenerator = new MockingDateGenerator();
+        attendanceManager = new AttendanceManager();
+    }
```

Comment:
```suggestion
    private final DateGenerator dateGenerator = new MockingDateGenerator();
    private final AttendanceManager attendanceManager = new AttendanceManager();
```
junit5의 기본 라이프사이클은 메소드당이기 때문에 굳이 beforeEach로 가변적으로 선언할 필요는 없어요.

https://junit.org/junit5/docs/current/user-guide/#writing-tests-test-instance-lifecycle

Reply:
헉 감사합니다! 몰랐던,, 사실입니다 😲